src/inatinqperf/benchmark/benchmark.py

```diff
@@ -195,10 +157,11 @@ def search(self, dataset: Dataset, vectordb: VectorDatabase, baseline_vectordb:
 
         logger.info("Performing search on baseline")
         with Profiler("search-baseline-FaissFlat") as p:
-            i0 = np.empty((q.shape[0], topk), dtype=float)
+            i0 = np.full((q.shape[0], topk), -1.0, dtype=float)
```

Comment:
Why `np.full`? We are populating the i0 matrix so we can initialize it as empty?